test/com/gu/contentapi/sanity/CODESuite.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.contentapi.sanity
+
+import com.gu.contentapi.sanity.support.{FakeAppSupport, DoNothing}
+import org.scalatest.Suites
+
+/**
+ * Nested suite of all the tests that should be run in the CODE environment
+ */
+class CODESuite extends Suites(
+  Seq(
+    new DraftR2ContentShouldAppearInPreviewTest,
+    new ReadComposerDraftInPreviewTest
```

Comment:
I think you need to update the README to reflect that CODE tests should be added to this class?